### DISCUSSION TITLE
TransitionLayerCount not working

### DISCUSSION THREAD

**Discussion by Wi419** [sn4k3/UVtools#902] [Q&A] [answered]
I tried converting the SL1 file to CTB versions 3, 4 and 5 and the GOO format. The converted file does not contain transition layers. UVtools skips the "TransitionLayerCount_4" entry in Prusa Slicer. The exposure of the base layers is set to 30 and the normal ones to 8. The number of base layers is 5. I am using UVtools version 4.3.2. Please advise what I may be doing wrong.

**Comment by sn4k3** [comment]
Can you share your SL1 file?

**Comment by Wi419** [comment]
There is my SL1 file: https://drive.google.com/file/d/1nRwX6nlMtc7UELK6rAL2AyUZ140oQtSX/view?usp=sharing
Github didn't let me attach it.

**Reply by sn4k3** [reply, marked as answer]
Nothing wrong with the file, it's a bug in the software. Will be fixed on next patch.
Meanwhile use UVtools suggestions to auto set transition layers.

**Reply by Wi419** [reply]
I just started to learn UVtools and thought it was my fault. Thank you very much for your help!
